plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/notifications/NotificationServiceInitializer.kt

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.notifications
+
+import com.intellij.ide.util.RunOnceUtil
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.startup.ProjectActivity
+
+internal class NotificationServiceInitializer : ProjectActivity {
+
+    override suspend fun execute(project: Project) {
+        val service = NotificationPollingService.getInstance()
+        ProcessNotificationsBase()
```

Comment:
why is this required?

Reply:
ProcessNotificationsBase needs to initialize somehow (after the pollingservice init and before the start polling begins) so that it's observer is able to be notified on startup. I tried some other ways of implementing, but this is what I found to work